python/rikai/spark/functions/vision.py

```diff
@@ -280,3 +281,22 @@ def spectrogram_image(
         format=image_format,
         **image_kwargs,
     )
+
+
+@udf(returnType=ArrayType(IntegerType()))
```

Comment:
let's move this to rikai.contrib.spark.vision maybe? how do you feel about putting functionality that's useful but not quite so common to most rikai users to the contrib section @eddyxu ?